pkg/instance/instance_test.go

```diff
@@ -30,10 +30,11 @@ func TestFuzzerCmd(t *testing.T) {
 	flagExecutor := flags.String("executor", "./syz-executor", "path to executor binary")
 	flagSignal := flags.Bool("cover", false, "collect feedback signals (coverage)")
 	flagSandbox := flags.String("sandbox", "none", "sandbox for fuzzing (none/setuid/namespace)")
+	flagSandboxArg := flags.Int("sandbox_arg", 0, "")
```

Comment:
See the comment at the top of the function.

Reply:
Is there is a test case for this? Or should I perform this scenario manually?

Reply:
What test case do you mean? This is a test case.